src/components/Navbar/Navbar.tsx

```diff
@@ -118,4 +117,13 @@ const Navbar = (props: navbarProps) => {
     )
 }
 
+// not needed?
+export async function getServerSideProps(context: any) {
+    return {
+        props: {
+            csrfToken: await getCsrfToken(context),
+        },
+    }
+}
+
```

Comment:
hah cause i wrapped this component in dynamic. If we want to keep this component as ssr we can probably use this value to control the UI state.

Reply:
i removed the csrftoken as the nonce as it was abit wonky guess we can just remove this